File: Python/celula.py
#!/usr/bin/python
# -*- coding: UTF-8 -*-
from citoesqueleto import citoesqueleto
from citoplasma import citoplasma
from nucleo import nucleo
from membranaCelular import membranaCelular
from reticuloEndoplasmatico import reticuloEndoplasmatico
from aparatoGolgi import aparatoGolgi
from organulos import organulos
from typing import List

class celula(object):
	def getCitoesqueletos(self) -> citoesqueleto:
		pass

	def setCitoesqueletos(self, aCitoesqueletos : citoesqueleto) -> None:
		pass

	def getCitoplasma(self) -> citoplasma:
		return self.___citoplasma

	def setCitoplasma(self, aCitoplasma : citoplasma) -> None:
		self.___citoplasma = aCitoplasma

	def getNucleo(self) -> nucleo:
		return self.___nucleo

	def setNucleo(self, aNucleo : nucleo) -> None:
		self.___nucleo = aNucleo

	def getMitocondria(self) -> mitocondria:
		return self.___mitocondria

	def setMitocondria(self, aMitocondria : mitocondria) -> None:
		self.___mitocondria = aMitocondria

	def getRibosoma(self) -> ribosoma:
		return self.___ribosoma

	def setRibosoma(self, aRibosoma : ribosoma) -> None:
		self.___ribosoma = aRibosoma

	def getMembranaCelular(self) -> membranaCelular:
		return self.___membranaCelular

	def setMembranaCelular(self, aMembranaCelular : membranaCelular) -> None:
		self.___membranaCelular = aMembranaCelular

	def getReticuloEndoplasmatico(self) -> reticuloEndoplasmatico:
		return self.___reticuloEndoplasmatico

	def setReticuloEndoplasmatico(self, aReticuloEndoplasmatico : reticuloEndoplasmatico) -> None:
		self.___reticuloEndoplasmatico = aReticuloEndoplasmatico

	def getAparatoGolgi(self) -> aparatoGolgi:
		return self.___aparatoGolgi

	def setAparatoGolgi(self, aAparatoGolgi : aparatoGolgi) -> None:
		self.___aparatoGolgi = aAparatoGolgi

	def __init__(self):
		self.___citoesqueletos : citoesqueleto = None
		self.___citoplasma : citoplasma = None
		self.___nucleo : nucleo = None
		self.___mitocondria : mitocondria = None
		self.___ribosoma : ribosoma = None
		self.___membranaCelular : membranaCelular = None
		self.___reticuloEndoplasmatico : reticuloEndoplasmatico = None
		self.___aparatoGolgi : aparatoGolgi = None
		self._unnamed_nucleo_ : nucleo = None
		self._unnamed_membranaCelular_ : membranaCelular = None
		self._unnamed_reticuloEndoplasmatico_ : reticuloEndoplasmatico = None
		self._unnamed_aparatoGolgi_ : aparatoGolgi = None
		self._unnamed_citoplasma_ : citoplasma = None
		self._unnamed_citoesqueleto_ : citoesqueleto = None
		self._unnamed_organulos_ : organulos = None

File: C#/celula.cs
using System;

public class Celula {
	private Citoesqueleto[] citoesqueletos;
	public Citoesqueleto[] Citoesqueletos {
		get {
			return citoesqueletos;
		}
		set {
			citoesqueletos = value;
		}
	}
	private Citoplasma citoplasma;
	private Nucleo nucleo;
	private Mitocondria mitocondria;
	private Ribosoma ribosoma;
	private MembranaCelular membranaCelular;
	private ReticuloEndoplasmatico reticuloEndoplasmatico;
	private AparatoGolgi aparatoGolgi;

	public Citoesqueleto GetCitoesqueletos() {
		return this.citoesqueletos;
	}
	public void SetCitoesqueletos(ref Citoesqueleto citoesqueletos) {
		this.citoesqueletos = citoesqueletos;
	}
	public Citoplasma GetCitoplasma() {
		return this.citoplasma;
	}
	public void SetCitoplasma(ref Citoplasma citoplasma) {
		this.citoplasma = citoplasma;
	}
	public Nucleo GetNucleo() {
		return this.nucleo;
	}
	public void SetNucleo(ref Nucleo nucleo) {
		this.nucleo = nucleo;
	}
	public Mitocondria GetMitocondria() {
		return this.mitocondria;
	}
	public void SetMitocondria(ref Mitocondria mitocondria) {
		this.mitocondria = mitocondria;
	}
	public Ribosoma GetRibosoma() {
		return this.ribosoma;
	}
	public void SetRibosoma(ref Ribosoma ribosoma) {
		this.ribosoma = ribosoma;
	}
	public MembranaCelular GetMembranaCelular() {
		return this.membranaCelular;
	}
	public void SetMembranaCelular(ref MembranaCelular membranaCelular) {
		this.membranaCelular = membranaCelular;
	}
	public ReticuloEndoplasmatico GetReticuloEndoplasmatico() {
		return this.reticuloEndoplasmatico;
	}
	public void SetReticuloEndoplasmatico(ref ReticuloEndoplasmatico reticuloEndoplasmatico) {
		this.reticuloEndoplasmatico = reticuloEndoplasmatico;
	}
	public AparatoGolgi GetAparatoGolgi() {
		return this.aparatoGolgi;
	}
	public void SetAparatoGolgi(ref AparatoGolgi aparatoGolgi) {
		this.aparatoGolgi = aparatoGolgi;
	}

	private Citoesqueleto citoesqueleto;
	private Organulos organulos;

}
